libclc/amdgcn-amdhsa/libspirv/math/acos.cl

```diff
@@ -0,0 +1,19 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include <spirv/spirv.h>
+#include <clcmacro.h>
```

Comment:
Sorting everywhere?

Reply:
Thanks for your suggestion, I will change it now.

Reply:
@keryell  Sorry to disturb you, I have modified it, please check if it is correct.

Reply:
> @keryell Sorry to disturb you, I have modified it, please check if it is correct.

Thanks! It looks quite better to figure out what was added.